### PR TITLE
Azure management containerinstance spot confidential public preview changes

### PR DESCRIPTION
**Pull request by fnuarnav**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by ArthurMa1978** [comment]
@fnuarnav the code you are updating is the old Track 1 Sdk that is being deprecated, please follow this [release process](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process) to update the new [Track 2 Sdk](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/containerinstance/Azure.ResourceManager.ContainerInstance).

File: sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/Models/ContainerGroup.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Azure.Management.ContainerInstance.Models
     /// A container group.
     /// </summary>
     [Rest.Serialization.JsonTransformation]
-    public partial class ContainerGroup : Resource
+    public partial class ContainerGroup
```
Comment (fengzhou-msft) [inline sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/Models/ContainerGroup.cs:24]: Is this expected?

Comment (diyellap) [reply sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/Models/ContainerGroup.cs:24]: This was autogenerated from new swagger spec. The build and tests passed, don't see any issues with it.

**Comment by diyellap** [comment]
[ArthurMa1978](https://github.com/ArthurMa1978) Track2 codegen is failing for ContainerInstance. We've sent an email to the SDK team detailing the errors. Track1 codegen and build were successful without any issues. We requested an exception for track1 release in the email.

**Comment by ArthurMa1978** [comment]
> [ArthurMa1978](https://github.com/ArthurMa1978) Arthur Ma FTE Track2 codegen is failing for ContainerInstance. We've sent an email to the SDK team detailing the errors. Track1 codegen and build were successful without any issues. We requested an exception for track1 release in the email.

Here is an issue opened https://github.com/Azure/azure-sdk-for-net/issues/33739 to track release Track 2 sdk.

**Pull request closed without merging**